products/microsoft_defender_for_endpoints.py

```diff
@@ -139,6 +143,9 @@ def process_search(self, tag: Tag, base_query: dict, query: str) -> None:
         
         query += f" {self.build_query(base_query)}" if base_query != {} else ''
 
+        if self._limit > 0 and 'limit' not in query:
+            query += f"| limit {str(self._limit)}"
```

Comment:
Added a space for query formatting consistency

```suggestion
            query += f" | limit {str(self._limit)}"
```